Languages/Java/questions.json

```diff
@@ -151,6 +151,33 @@
       "trueOrFalse": false,
       "shortAns": "It is a problem that occurs while running the program",
       "tag": "Java"
+    },
+    {
+      "id": 17,
+      "question": "What is a key difference between Error and Exception?",
+      "details": "An irrecoverable state of the system is caused by an error while the system goes to a recoverable state when an exception is thrown.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "It is an error if the system cannot recover, while an exception if system resumes normal mode of operation.",
+      "tag": "Exception"
+    },
+    {
+      "id": 18,
+      "question": "What is the difference between throw and throws in Java?",
+      "details": "The throw keyword is used to throw an exception explicitly while the throws keyword can be used to declare multiple exceptions, separated by a comma.",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "throw is used in a function, throws is used in method signature",
+      "tag": "Exception"
+    },
+    {
+      "id": 19,
+      "question": "In Java 11, what would be the output of this line: (var s1, String y) -> s1 + y;",
+      "details": "cannot mix 'var' and explicitly-typed parameters, Java 11 introduced Local-Variable Syntax for Lambda Parameters, but mixing is not allowed",
+      "questionType": "multi",
+      "trueOrFalse": false,
+      "shortAns": "cannot mix 'var' and explicitly-typed parameters",
```

Comment:
Sorry but this type of question wont work with this multiple choice quiz. Please do a different question, use the android/question.json for reference.